File: packages/firestore/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/firestore/classes/RemoveSnapshotListenerOptions.java
package io.capawesome.capacitorjs.plugins.firebase.firestore.classes;

public class RemoveSnapshotListenerOptions {
    private String callbackId;

    public RemoveSnapshotListenerOptions(String callbackId) {
        this.callbackId = callbackId;
    }

    public String getCallbackId() {
        return callbackId;
    }
}

File: packages/firestore/ios/Plugin/Classes/Options/RemoveSnapshotListenerOptions.swift
import Foundation

@objc public class RemoveSnapshotListenerOptions: NSObject {
    private var callbackId: String

    init(callbackId: String) {
        self.callbackId = callbackId
    }

    func getCallbackId() -> String {
        return self.callbackId
    }
}
